pkg/storage/unified/sql/blob.go

```diff
@@ -57,6 +58,7 @@ func (b *backend) PutResourceBlob(ctx context.Context, req *resource.PutBlobRequ
 	err = b.db.WithTx(ctx, ReadCommitted, func(ctx context.Context, tx db.Tx) error {
 		_, err := dbutil.Exec(ctx, tx, sqlResourceBlobInsert, sqlResourceBlobInsertRequest{
 			SQLTemplate: sqltemplate.New(b.dialect),
+			Now:         time.Now(),
```

Comment:
🤦🏻 previously everything in SQL had a zero timestamp

<img width="857" alt="image" src="https://github.com/user-attachments/assets/21020c28-4257-4afd-9621-b9e9c4392c66" />

Reply:
Perhaps this could be its own PR that would include the test too?